### PR TITLE
Rework custom defaults in dxLoadPanel, dxTabPanel & dxDateBox

### PR DESCRIPTION
**Pull request by KsyushaM**

### CODE REVIEW

File: js/ui/date_box/ui.date_box.js [hunk 1]
```diff
@@ -295,6 +295,12 @@ var DateBox = DropDownEditor.inherit({
             {
                 device: { platform: "ios" },
                 options: {
+                    /**
+                    * @name dxDateBoxOptions_pickerType
+                    * @publicName pickerType
+                    * @default 'native'
```
Comment (TikhonovaTatyana) [inline js/ui/date_box/ui.date_box.js]: In doc comment, the platform is not mentioned
@default 'native' @for iOS 
same for android

Comment (DokaRus) [inline js/ui/date_box/ui.date_box.js]: @TikhonovaTatyana We need specify a device here, right?
Like below
```
* @default 'native' @for iOS

File: js/ui/date_box/ui.date_box.js [hunk 2]
```diff
@@ -303,6 +309,12 @@ var DateBox = DropDownEditor.inherit({
                     return device.platform === "android";
                 },
                 options: {
+                    /**
+                    * @name dxDateBoxOptions_pickerType
+                    * @publicName pickerType
+                    * @default 'native'
```
Comment (DokaRus) [inline js/ui/date_box/ui.date_box.js]: @TikhonovaTatyana And there something like

```
* @default 'native' @for Android

**Review by DokaRus — CHANGES_REQUESTED** [review]

**Review by lukyanovas — DISMISSED** [review]
LGTM

**Review by DokaRus — DISMISSED** [review]
LGTM

**Review by lukyanovas — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**